.github/workflows/main.yml

```diff
@@ -0,0 +1,23 @@
+on: [push]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    name: lint
+    steps:
+      - name: checkout
+        uses: actions/checkout@v1.1.0
+      - name: eclint-action
+        uses: greut/eclint-action@v0.1.0
+      - name: golangci-lint
+        uses: ./.github/actions/golangci-lint
+  test:
+    runs-on: ubuntu-latest
+    name: test
+    steps:
+      - name: checkout
+        uses: actions/checkout@v1.1.0
+      - name: go test
+        uses: ./.github/actions/go-test
+      - name: core test
+        uses: ./.github/actions/core-test
```

Comment:
Both approaches are valid, but I'd personally create different workflows instead of having a single workflow calling multiple tasks.

Feel free to keep as is if you prefer, though. 🙂